Benchmarks/BenchmarkReporter.swift

```diff
@@ -0,0 +1,127 @@
+// Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Foundation
+
+/// Protocol that defines how are the benchmark results printed to the stdout. 
+protocol BenchmarkReporter {
+
+    /// Print benchmark results to stdout.
+    func printResults(_ result: BenchmarkResults)
+
+    /// Print benchmark default settings to stdout.
+    func printDefaults()
```

Comment:
For consistency with the `BenchmarkConfiguration` naming suggestion below:
```suggestion
    func printBenchmarkDefaultConfigurations()
```

Reply:
Edit: resolving this comment in favor of some bigger code refactoring! https://github.com/tensorflow/swift-models/pull/230#pullrequestreview-322493790